src/updates/node/installer.ts

```diff
@@ -0,0 +1,38 @@
+import { exec } from 'child_process';
+import * as semver from 'semver';
+
+export function checkNodeInstalled(): boolean {
```

Comment:
Keep similar to the other products so we have a consistent interface, i.e return the version if installed or undefined if not.
```suggestion
export function checkInstalledVersion(): string|undefined {
```